style-guides/scss.md

```diff
@@ -0,0 +1,123 @@
+<script>{
+	"title": "Chassis SCSS Style Guide"
+}</script>
+
+This page outlines the style guide for SCSS for the jQuery Chassis project.
+
+## Comments
+
+SCSS comments should follow the valid CSS comment guidelines outlined in the jQuery contribute CSS style guide.
```

Comment:
We actually decided against this a while back we should use which ever style makes sense. Comments which will lose context when converted to css. ( like variables ) should use `//` so that they will not end up in the final css with no context or rules near them that make any sense.